net.rpdev.OpenTodoList.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.rpdev.OpenTodoList.desktop</id>
+  <name>OpenTodoList</name>
+  <project_license>GPL-3.0+</project_license>
+  <summary>Todo list and note taking application</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <description>
+    <p>OpenTodoList is an todo list and note taking application. Create todo
+    lists, notes and put other resources, arranging them in libraries.
+    These libraries can be kept locally or synchronized with services
+    such as NextCloud or ownCloud to sync them across your devices.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>A library</caption>
+      <image>https://opentodolist.rpdev.net/images/v3.7/material-1.png</image>
+    </screenshot>
+  </screenshots>
```

Comment:
You should add `<launchable type="destkop-id">net.rpdev.OpenTodoList.desktop</launchable>